scss/detail.scss

```diff
@@ -7,12 +7,27 @@
   align-items: center;
   
   .cover-text-container {
+    position: relative;
     display: flex;
     height: 416px;
     width: 900px;
     background-color: $white;
     box-shadow: 4px 4px 10px 0 $black-rgba;
 
+    .return-button {
+      display: flex;
```

Comment:
Since ".return-button" element is not containing any element this rule can be removed